client/src/App.js

```diff
@@ -89,15 +89,12 @@ class App extends Component {
         <div hidden="true" id="new-project-form">
           <form onSubmit={this.submitNewProject}> 
             <h2> Submit a new project</h2>
-            <label for="form-requester">Requester Name</label>
-            <input name="form-requester" id="form-requester"></input>
-            <label for="form-title">Title of Project</label>
-            <input name="form-title" id="form-title"></input>
-            <label for="form-content">tell us about the project</label>
+            <label for="form-requester">Requester name: <input name="form-requester" id="form-requester"></input></label>
+            <label for="form-title">Title of project: <input name="form-title" id="form-title"></input></label>
+            <label for="form-content">Tell us about the project</label>
             <textarea name="form-content" id="form-content"></textarea>
-            <label for="form-url">add a link to more information</label>
-            <input name="form-url" id="form-url"></input>
-            <input type="submit"/>
+            <label for="form-url">Add a link to more information: <input name="form-url" id="form-url"></input></label>
```

Comment:
Maybe this could be `type="url"`, like [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url)? (There's also an [email type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) that we might want to use for the Requester? Or maybe not...)